src/py42/modules/securitydata.py

```diff
@@ -270,6 +343,16 @@ def _get_plans_in_node(destination, node):
     }
 
 
+def _parse_file_location_response(response):
+
+    for location in response[u"locations"]:
```

Comment:
Any chance of response not having `locations`? Is this is a good place to fail if that is the good or should we just treat it the file not having a location?

Reply:
e.g. raise a FileNotFound exception or something